handbook/product/design/research/index.md

```diff
@@ -2,18 +2,57 @@
-- **External user interviews.** Existing and prospective customers, participants we find through outreach on social media, and other sources are all great participants for research activities.
-- **Hallway testing.** When we want to test our ideas using interactive prototypes or even in the product itself, our own team members make great participants. (However, we need to remember that our own team members are generally very experienced in the area of code search, which results in implicit bias that we must account for in synthesis.)
-- **Unmoderated user testing.** When unmoderated user testing is valuable we use [usertesting.com](../tools/index.md#product-design-and-research) to conduct unmoderated user research with a pool of participants sourced by usertesting.com.
-- **Moderated user testing.** When moderated user testing is valuable, we use [LookBack](../tools/index.md#product-design-and-research) to conduct sessions. With LookBack, other team members can observe and take notes in real time without appearing to the participant, which makes for a more positive participation experience.
-- **Competitive and analogous research.** It's worthwhile to see how other tools and platforms are solving parallel problems. We usually keep screenshots and references in artifacts like RFCs and in project Figma files.
-- **Existing artifact analysis.** Our users are passionate and provide excellent, regular feedback that we keep in tools like [Productboard](https://sourcegraph.productboard.com/).
+### Hallway testing
+
+When we want to test our ideas using interactive prototypes or even in the product itself, our own team members make great participants. (However, we need to remember that our own team members are generally very experienced in the area of code search, which results in implicit bias that we must account for in synthesis.)
+
+### Unmoderated user testing
+
+When unmoderated user testing is valuable we use [usertesting.com](../tools/index.md#product-design-and-research) to conduct unmoderated user research with a pool of participants sourced by usertesting.com.
+
+> [Read the Sourcegraph guide for conducting research with usertesting.com](./user-testing-com.md)
+
+### Moderated user testing
+
+When moderated user testing is valuable, we use [LookBack](../tools/index.md#product-design-and-research) to conduct sessions. With LookBack, other team members can observe and take notes in real time without appearing to the participant, which makes for a more positive participation experience.
+
+### Competitive and analogous research
+
+It's worthwhile to see how other tools and platforms are solving parallel problems. We usually keep screenshots and references in artifacts like RFCs and in project Figma files.
```

Comment:
```suggestion
It can be beneficial to see how other tools and platforms are solving parallel problems. We usually collect and store screenshots and references in artifacts like RFCs and the Research layer in Figma projects. We also have [Styleboards in Figma](https://www.figma.com/files/project/10712517/Styleboards?fuid=1011662758768504201) for collecting inspiration for product areas. 
```